app/views/user_notifier/send_confirmation.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Order #<%= @order.id %></h1>
+<h3>You purchased:</h3>
+<br>
+<% @order.line_items.each do |line_item| %>
+  <h4><%= line_item.item.title %></h4>
+  <h5><%= line_item.item.description %></h5>
+  <h5>Quantity: <%= line_item.quantity %></h5>
+  <h5>Subtotal: <%= "$#{format("%.2f", line_item.subtotal/100.0)}" %></h5>
```

Comment:
actually I cant figure it out because mailers only have access to their own helper methods, which are not the same as those in ApplicationController.